### PR TITLE
PR for 2.3 release 

### PR DESCRIPTION
**Pull request by jfy133**
This PR is for 2.3 - "Aalen".

It includes a couple of minor bug fixes, and also additional functionality request by reviewers of the nf-core manuscript.

Functionality includes
- Low-sequence complexity read filtering prior metagenomic screening
- mapDamage rescaling of BAM files

Fixed issues
- DeDup nullPointerException fix (version bump)
- Fixed AWS stress test a bit more
- Fixed some mistakes in schema regarding dropdowns
- Fixed a (semi-)major bug where filtering options using circularmapper was flipped  

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
 - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/eager/tree/master/.github/CONTRIBUTING.md)
 - [x] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,10 +22,42 @@
...
+    ```bash
+    nextflow run nf-core/eager -profile test,<docker/singularity/podman/conda/institute>
+    ```
+
+    > Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
+
+4. Start running your own analysis!
+
+    ```bash
+    nextflow run nf-core/eager -profile <docker/singularity/conda> --input '*_R{1,2}.fastq.gz' --fasta '<your_reference>.fasta'
```
Comment (MaxUlysse) [inline README.md:45]: we have `nextflow run nf-core/eager -profile test,<docker/singularity/podman/conda/institute` in the previous example, should we have something similar here? (or maybe it's this way in the template actually, and that's what need fixing...)

File: bin/scrape_software_versions.py [hunk 2]
```diff
@@ -71,6 +73,9 @@
 results['kraken'] = '<span style="color:#999999;\">N/A</span>'
 results['maltextract'] = '<span style="color:#999999;\">N/A</span>'
 results['eigenstrat_snp_coverage'] = '<span style="color:#999999;\">N/A</span>'
+results['mapDamage2'] = '<span style="color:#999999;\">N/A</span>'
+results['bbduk'] = '<span style="color:#999999;\">N/A</span>'
+
```
Comment (MaxUlysse) [inline bin/scrape_software_versions.py:78]:
```suggestion
```

File: main.nf [hunk 3]
```diff
@@ -808,6 +819,8 @@ Channel.from(summary.collect{ [it.key, it.value] })
     """.stripIndent() }
     .set { ch_workflow_summary }
 
+log.info "Schaffa, Schaffa, Genome Baua!"
```
Comment (MaxUlysse) [inline main.nf:822]: ?

File: main.nf [hunk 4]
```diff
@@ -2443,7 +2487,7 @@ process qualimap {
     script:
     def snpcap = params.snpcapture_bed != '' ? "-gff ${params.snpcapture_bed}" : ''
     """
-    qualimap bamqc -bam $bam -nt ${task.cpus} -outdir . -outformat "HTML" ${snpcap}
+    qualimap bamqc -bam $bam -nt ${task.cpus} -outdir . -outformat "HTML" ${snpcap} --java-mem-size=${task.memory.toGiga()}G
```
Comment (MaxUlysse) [inline main.nf:2490]: Did that solve your issues with qualimap?
(asking for a friend)

**Review by MaxUlysse — APPROVED** [review]
Looks good to me ;-)

File: main.nf (hunk 3 above)
Comment (apeltzer) [reply main.nf:822]: This is an easter egg we had in EAGERv1 and James wanted to make this available in the new version too as people got used to it ;-) Swabian German for "Work, Work, Build a genome!" (+-, hard to 100% transcribe)

File: main.nf (hunk 4 above)
Comment (apeltzer) [reply main.nf:2490]: Pinging @maxibor who added this - yes, I think so 🤔

**Review by apeltzer — APPROVED** [review]
LGTM :-)

File: main.nf (hunk 3 above)
Comment (Zethson) [reply main.nf:822]: @apeltzer @MaxUlysse https://www.youtube.com/watch?v=831QzzTuCjA

Keep the easter egg please :)

File: README.md (hunk 1 above)
Comment (jfy133) [reply README.md:45]: No that was my mistake!

**Pull request merged**